### PR TITLE
in_windows_exporter_metrics: Add document for process metrics

### PR DESCRIPTION
**Pull request by cosmo0920**
This PR is corresponding to https://github.com/fluent/fluent-bit/pull/7860.

### CODE REVIEW

File: pipeline/inputs/windows-exporter-metrics.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ The initial release of Windows Exporter Metrics contains a single collector avai
 | we.service.where           | Specify the WHERE clause for retrieving service metrics.  | `NULL`   |
 | we.service.include         | Specify the key value pairs for the include condition for the WHERE clause of service metrics. | `NULL`   |
 | we.service.exclude         | Specify the key value pairs for the exclude condition for the WHERE clause of service metrics. | `NULL`   |
+| we.process.allow\_process\_regex | Specify the regex for process metrics to allow collection of. Collect all by default. | "/.+/"    |
```
Comment (patrick-stephens) [inline pipeline/inputs/windows-exporter-metrics.md:27]: Probably just reword slightly:
```suggestion
| we.process.allow\_process\_regex | Specify the regex covering the process metrics to  collect. Collect all by default. | "/.+/"    |
```

Comment (cosmo0920) [reply pipeline/inputs/windows-exporter-metrics.md]: Ah, good idea. process is able to use verb so your suggestion makes clear its meaning.

**Review by patrick-stephens — APPROVED** [review]
Looks good but need to sort conflicts

**Comment by cosmo0920** [comment]
I sorted the conflicts.

**Pull request merged**